Controllers/AttendanceController.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using WsuTriangle.AttendanceTracker.Models;
+
+namespace WsuTriangle.AttendanceTracker.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class AttendanceController(AppDbContext db) : ControllerBase
```

Comment:
when you get to it, this will force authorization on the entire controller. don't need to do it individually unless there is a specific role
```suggestion
[Authorize] 
public class AttendanceController(AppDbContext db) : ControllerBase
```